src/content/docs/tailwind-case-study/tailwind-case-study.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Tailwind css
+author: Bavy Ado <Bavy89>
+tags: tailwind-css, frontend, css-framework, case-study
+---
+
```

Comment:
@Bavy89 Good job for implementing the feedback. However, now you have a duplicate. You need to remove the old one. The new one `[src/content/docs/frameworks/css/tailwind-case-study.md](https://github.com/NoroffFEU/development-platforms/pull/291/files#diff-caf64c6ec57e6a2da2e16c363b29a405b78d43a312deb01a1a482518e91ab174)` looks good to me.